build.gradle

```diff
@@ -43,32 +43,35 @@ ext {
             aplicationId     : "ru.startandroid.organizer",
             versionCode      : 1,
             versionName      : "1.0",
-
-            minSdkVersion    : 20,
+            minSdkVersion    : 21,
             targetSdkVersion : 26,
             compileSdkVersion: 28,
+
     ]
 
     versions = [
-            androidx: "1.1.0-alpha04",
-            constraintLayout: '2.0.0-beta1',
-            junit: '4.13-beta-3',
-            espresso: '3.2.0-beta01',
-            androidxtest: '1.2.0-beta01',
-            dagger2: '2.16',
-            firebase: '16.0.9',
-            crashlytics: '2.10.0',
-            utils: "1.0.0-beta01",
-            recyclerview: "1.0.0-beta01",
-            room: '2.1.0-beta01',
-            gson: "2.8.5",
-            rxjava2: '2.2.8',
-            rxjava2adapter: '2.5.0',
-            rxandroid: '2.1.1',
-            cardview: "1.0.0",
-            retrofit: "2.5.0",
-            logginginterceptor:"3.12.0",
-            workmanager: "2.0.1"
+            androidx          : "1.1.0-alpha04",
+            constraintLayout  : '2.0.0-beta1',
+            junit             : '4.13-beta-3',
+            espresso          : '3.2.0-beta01',
+            androidxtest      : '1.2.0-beta01',
+            dagger2           : '2.16',
+            firebase          : '16.0.9',
+            crashlytics       : '2.10.0',
+            utils             : "1.0.0-beta01",
+            recyclerview      : "1.0.0-beta01",
+            room              : '2.1.0-beta01',
+            gson              : "2.8.5",
+            rxjava2           : '2.2.8',
+            rxjava2adapter    : '2.5.0',
+            rxandroid         : '2.1.1',
+            cardview          : "1.0.0",
+            retrofit          : "2.5.0",
+            logginginterceptor: "3.12.0",
+            workmanager       : "2.0.1",
+            material          : "1.0.0",
+            appcompat         : "1.0.2",
+            constraintlayout  : "1.1.3"
```

Comment:
Can be removed. We already have it

Reply:
Done

Reply:
Sorry, can't get what can be removed. We use versions' names further in gradle dependencies...

Reply:
Done